src/app/src/main/kotlin/org/imperial/mrc/hint/security/OneTimeTokenGenerator.kt

```diff
@@ -0,0 +1,62 @@
+package org.imperial.mrc.hint.security
+
+import org.imperial.mrc.hint.AppProperties
+import org.pac4j.core.profile.CommonProfile
+import org.springframework.context.annotation.Configuration
+import org.pac4j.jwt.config.signature.RSASignatureConfiguration
+import org.pac4j.jwt.profile.JwtGenerator
+//import org.vaccineimpact.api.app.repositories.TokenRepository
+//import org.vaccineimpact.api.models.permissions.ReifiedPermission
+//import org.vaccineimpact.api.models.permissions.ReifiedRole
+//import org.vaccineimpact.api.security.InternalUser
+//import org.vaccineimpact.api.security.KeyHelper
+//import org.vaccineimpact.api.security.WebTokenHelper
+//import org.vaccineimpact.api.security.deflated
+import java.time.Duration
+import java.security.KeyPair
+import java.security.SecureRandom
+import java.time.Instant
+import java.util.*
+
+enum class TokenType
+{
+    ONETIME
+}
+
+@Configuration
+open class OneTimeTokenGenerator(
```

Comment:
This class combines functionality from montagu api's OneTimeTokenGenerator and WebTokenHelper, as I figured neither will have to be as complex as in montagu